app/models/appointment.rb

```diff
@@ -1,4 +1,14 @@
 class Appointment < ActiveRecord::Base
+  enum status: %i(
+    pending
+    completed
+    no_show
+    ineligible_age
+    ineligible_pension_type
+    cancelled_by_customer
+    cancelled_by_pension_wise
+  )
```

Comment:
do we really want to store status in the DB as an integer with a special meaning that is only defined here? 

can't we just store it as a text field?

 or if we have to use integers use a lookup table.

On a plus side this approach means that if you change the ordering in this list you break historical data. :)